cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -470,7 +214,76 @@ def close(self):
+    def create_image_if_not_exists(self, install_path):
+        if self.task_created:
+            return
+        _, image_name = install_path.split('/')
+        with lock.NamedLock("upload-image-%s" % self.taskUuid):
+            if not self.task_created:
+                rbd.RBD().create(self.ioctx, image_name, self.expectedSize)
+                self.task_created = True
+
+    @staticmethod
+    def check_capacity(required_size):
+        total, avail, _ = _get_capacity()
```

Ruff (0.14.0)

Unpacked variable `total` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)